smart_contract/executor/src/lib.rs

```diff
@@ -222,16 +288,22 @@ pub mod prelude {
     pub use alloc::vec::Vec;
 
     pub use iroha_executor_derive::{
-        entrypoint, Constructor, Token, Validate, ValidateEntrypoints, ValidateGrantRevoke, Visit,
+        entrypoint, Constructor, Permission, Validate, ValidateEntrypoints, ValidateGrantRevoke,
+        Visit,
+    };
+    pub use iroha_smart_contract::prelude::{
+        Parameter as ParameterObject, Permission as PermissionObject, *,
```

Comment:
I'm not a fan of this renaming to `*Object`. If we have to do this I'd prefer we do it for the trait

Reply:
Done.